File: Share/src/main/java/com/buguagaoshu/share/domain/DiskMessage.java
package com.buguagaoshu.share.domain;

import jakarta.persistence.Entity;
import lombok.Data;
import jakarta.persistence.Id;

/**
 * @author Pu Zhiwei {@literal puzhiweipuzhiwei@foxmail.com}
 * create          2020-07-08 20:09
 */
@Data
@Entity
public class DiskMessage {
    @Id
    private Integer id;

    private Long userDisk;

    private Long availableDisk;

    private Long uploadFileMax;

    private Integer editType;

}

File: Go/internal/model/disk_message.go
package model

// DiskMessage 对齐 Spring com.buguagaoshu.share.domain.DiskMessage（表 disk_message，单例 id=1）
type DiskMessage struct {
	ID             int   `gorm:"primaryKey;column:id" json:"id"`
	UserDisk       *int64 `gorm:"column:user_disk" json:"userDisk"`
	AvailableDisk  *int64 `gorm:"column:available_disk" json:"availableDisk"`
	UploadFileMax  *int64 `gorm:"column:upload_file_max" json:"uploadFileMax"`
	EditType       *int   `gorm:"column:edit_type" json:"editType"`
}

func (DiskMessage) TableName() string { return "disk_message" }
